src/common/commonutils/FileUtils.c

```diff
@@ -20,15 +22,12 @@ char* LoadStringFromFile(const char* fileName, bool stopAtEol, void* log)
     {
         if (LockFile(file, log))
         {
-            fseek(file, 0, SEEK_END);
-            fileSize = ftell(file);
-            fseek(file, 0, SEEK_SET);
-
-            string = (char*)malloc(fileSize + 1);
-            if (string)
+            if (NULL != (string = (char*)malloc(initialSize)))

```

Comment:
The `(char *)` cast is present here but not on line 45 with the `realloc` call. I'd say drop it here for consistency (it's not necessary).

Reply:
Well spotted, thanks! Will do